lib/Chunk.js

```diff
@@ -278,20 +444,33 @@ class Chunk {
 		return true;
 	}
 
+	/**
+	 *
+	 * @param {number} size the size
+	 * @param {Object} options the options passed in
+	 * @return {number} the multiplier returned
+	 */
 	addMultiplierAndOverhead(size, options) {
 		const overhead =
-			typeof options.chunkOverhead === "number" ? options.chunkOverhead : 10000;
+			options.chunkOverhead === "number" ? options.chunkOverhead : 10000;
```

Comment:
This removal of typeof looks incorrect. Is it?

Reply:
oh yup this is the cause, thanks for seeing that! I hadn't gotten back to this yet. Thanks! Correcting.